flixel/util/FlxTimer.hx

```diff
@@ -273,4 +273,14 @@ class FlxTimerManager extends FlxBasic
 	{
 		FlxArrayUtil.clearArray(_timers);
 	}
-}
\ No newline at end of file
+	
+	/**
+	 * Calls the onComplete function of all timers, then removes them from the timer manager.
+	 */
+	public function completeAll():Void
+	{
+		for (timer in _timers)
+			timer.onComplete(timer);
```

Comment:
Ah!  That makes sense.  Thanks for explaining.  Maybe a while loop could work here.